lib/src/wallet/wallet_page.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/material.dart';
+
+class WalletPage extends StatelessWidget {
```

Comment:
If this is the page conating the ticket, consider naming it `TicketPage`, because there's another page for wallet.
Add both pages to the `BottomNavBar` so that the items will be:

 - TicketPage
 - WalletPage
 - AccountPage